flows/resumes/base.go

```diff
@@ -52,7 +51,7 @@ func (r *baseResume) Contact() *flows.Contact       { return r.contact }
 func (r *baseResume) ResumedOn() time.Time          { return r.resumedOn }
 
 // Apply applies our state changes and saves any events to the run
-func (r *baseResume) Apply(run flows.FlowRun, logEvent flows.EventCallback) error {
+func (r *baseResume) Apply(run flows.FlowRun, logEvent flows.EventCallback) {
```

Comment:
kinda unrelated but I noticed that there's no code paths now where Resume apply methods can error